Rootlog_V2.md

```diff
@@ -748,9 +748,9 @@ Common sequences of multiple actions in a row.
 - `Sp->9/(w+Ew+Lw)9->` *Furious Protector comes into town.*
 - `#bankO->/w->$` *The Riverfolk use Export to gain a warrior in Payments. Note: Don't use the "craft" notation in this case, since the card isn't added to Crafted Improvements and the item isn't added to the faction board.*
 - `%r->e/CXO12/(2Cw+3Ow)12->/$_O+$_C->h/++3` *The Vagrant Vagabond instigates a battle between the Marquise and the Riverfolk.*
-- `t_t7_10_11->10/t_t10^2/#faith$_2->` - *The Keepers delve a relic (possibly foolishly).*
+- `t_t7_10_11->10/t10^t_2_t/#faith$_2->` - *The Keepers delve a relic (possibly foolishly).*
```

Comment:
What notation is `t_2_t`? The notation for flipping a relic is `<relic token><Clearing>^<Number>`, so I would expect only a number to be on the right side of the `^`, unless I've forgotten some combination of notations here?

Reply:
This is to be more in line with the Corvids' flip notation.